nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -967,6 +967,58 @@ public void supportObjectsIsNull() {
         .doTest();
   }
 
+  @Test
+  public void supportAssertThatIsNotNull_String() {
+    compilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  private void foo(@Nullable String s) {",
+            "    com.google.common.truth.Truth.assertThat(s).isNotNull();",
+            "    s.toString();",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  public void supportAssertThatIsNotNull_Object() {
+    compilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.lang.Object;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  private void foo(@Nullable Object o) {",
+            "    com.google.common.truth.Truth.assertThat(o).isNotNull();",
+            "    o.toString();",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  public void supportAssertThatIsNotNull_Array() {
+    compilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  private void foo(@Nullable int[] i) {",
```

Comment:
Let's remove this one, actually. NullAway support for `@Nullable` arrays is sort of weird (we'd eventually like it to be able to reason about "an array of nullable X" but right now it considers this syntax to mean "a nullable array of non-null X", also "an array of nullable ints" makes no sense :) )

Reply:
Removed.